### PR TITLE
Akka Typed Add example of how to java `getSelf` for persistence typed #27061

### PR DESCRIPTION
**Pull request by helena**
https://github.com/akka/akka/issues/27061 

## Purpose
Makes it easier to find how to access `self` from typed java API.
 
## Background Context
Motivated by a user question

## Original PR 
Closed https://github.com/akka/akka/pull/27165 
Used @raboof's suggestions which made more sense than on the ticket, and I found a better location for the addition.

### CODE REVIEW

File: akka-persistence-typed/src/test/java/jdocs/akka/persistence/typed/BasicPersistentBehaviorTest.java [hunk 1]
```diff
@@ -4,29 +4,22 @@
...
+import akka.persistence.typed.*;
+import akka.persistence.typed.javadsl.*;
 
 import java.time.Duration;
 import java.util.ArrayList;
 import java.util.List;
 import java.util.Set;
 
+// #behavior
+// #behavior
```
Comment (raboof) [inline akka-persistence-typed/src/test/java/jdocs/akka/persistence/typed/BasicPersistentBehaviorTest.java]: Looks like something (I think IntelliJ Organize Imports) removed the imports from the sample?

**Review by raboof — APPROVED** [review]

File: akka-persistence-typed/src/test/java/jdocs/akka/persistence/typed/BasicPersistentBehaviorTest.java (hunk 1 above)
Comment (helena) [reply akka-persistence-typed/src/test/java/jdocs/akka/persistence/typed/BasicPersistentBehaviorTest.java]: oh evil, I'll check thanks!

Comment (helena) [reply akka-persistence-typed/src/test/java/jdocs/akka/persistence/typed/BasicPersistentBehaviorTest.java]: Fixed 👍

**Review by chbatey — APPROVED** [review]
LGTM

**Pull request merged**